galaxy/Dockerfile

```diff
@@ -222,7 +223,7 @@ RUN mkdir -p /shed_tools $EXPORT_DIR/ftp/ \
     && python /usr/local/bin/setup_postgresql.py --dbuser galaxy --dbpassword galaxy --db-name galaxy --dbpath $PG_DATA_DIR_DEFAULT --dbversion $PG_VERSION \
     && service postgresql start \
     && service postgresql stop \
-    && apt-get autoremove -y && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && rm -rf ~/.cache/ \
+    && apt-get autoremove -y make g++ gcc && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && rm -rf ~/.cache/ \
```

Comment:
It looks like pysam is expecting `make` (and likely gcc) to exist still. Although I would have expected the Galaxy environment to have a precompiled build of pysam already.